docs/ops/generation/EyeLike_9.md

```diff
@@ -0,0 +1,142 @@
+## EyeLike <a name="EyeLike"></a> {#openvino_docs_ops_generation_EyeLike_9}
+
+**Versioned name**: *EyeLike-9*
```

Comment:
Please let us call this operation as `Eye` because most of frameworks (TF, PyTorch, MXNet) have this name.
`EyeLike` is from ONNX and uses a shape of input tensor (a bit different semantic).